consensus/istanbul/backend/backend.go

```diff
@@ -110,10 +110,11 @@ func New(config *istanbul.Config, db ethdb.Database) consensus.Istanbul {
 		selfRecentMessages:                 selfRecentMessages,
 		announceThreadWg:                   new(sync.WaitGroup),
 		announceThreadQuit:                 make(chan struct{}),
+		generateAndGossipAnnounceCh:        make(chan struct{}, 1),
 		updateAnnounceVersionCh:            make(chan struct{}),
 		updateAnnounceVersionCompleteCh:    make(chan struct{}),
 		lastAnnounceGossiped:               make(map[common.Address]time.Time),
-		cachedAnnounceMsgs:                 make(map[common.Address]*announceMsgCachedEntry),
+		lastSignedAnnounceVersionsGossiped: make(map[common.Address]time.Time),
```

Comment:
Nit: Consider renaming this to something like "signedAnnounceVersionsRegossipCooldown" to be more descriptive (and the same comment for the other similar variable on line 201)